unit_tests/speech/test_operation.py

```diff
@@ -84,8 +84,10 @@ def test_poll_complete(self):
 class _Connection(object):
     def __init__(self, response):
         self.response = response
+        self._requested = []
 
     def api_request(self, method, path):
+        self._requested.append({'method': method, 'path': path})
```

Comment:
Hmm, doesn't look like the tests are making any assertion about the value of `connection._requested`.